src/graph/create-property.ts

```diff
@@ -0,0 +1,62 @@
+import { PROPERTY, RELATION_VALUE_RELATIONSHIP_TYPE, SCHEMA_TYPE, VALUE_TYPE_PROPERTY } from '../core/ids/system.js';
+import { generate } from '../id.js';
+import { Relation } from '../relation.js';
+import type { DefaultProperties, Op, ValueType } from '../types.js';
+import { createDefaultProperties } from './helpers/create-default-properties.js';
+
+type Params = DefaultProperties &
+  ({ type: ValueType } | { type: 'RELATION'; properties?: Array<string>; relationValueTypes?: Array<string> });
+
+/**
+ * Creates a property with the given name, description, cover, and type.
+ */
+export const createProperty = (params: Params) => {
+  const { name, description, cover } = params;
+  const entityId = generate();
+  const ops: Op[] = [];
+
+  ops.push(...createDefaultProperties({ entityId, name, description, cover }));
+
+  // add "Property" to property "Types"
+  const typesRelationOp = Relation.make({
+    fromId: entityId,
+    relationTypeId: PROPERTY,
+    toId: SCHEMA_TYPE,
+  });
+  ops.push(typesRelationOp);
+
+  if (params.type === 'RELATION') {
+    // add the provided properties to property "Properties"
```

Comment:
Might need to double check with Yaniv, but I think if it's a property meant to be used for Relations, then we should add `Types -> Type` to it.

```ts
const typeOps = Relation.make({
    fromId: entityId,
    relationTypeId: TYPES_PROPERTY,
    toId: SCHEMA_TYPE
})
```

Reply:
Notice here how `Works at` has both Property _and_ Type

Reply:
You are right, added it